app/javascript/dashboard/routes/dashboard/settings/inbox/components/AddCampaign.vue

```diff
@@ -1,132 +1,124 @@
 <template>
-  <modal :show.sync="show" :on-close="onClose">
-    <div class="column content-box">
-      <woot-modal-header
-        :header-title="$t('CAMPAIGN.ADD.TITLE')"
-        :header-content="$t('CAMPAIGN.ADD.DESC')"
-      />
-      <form class="row" @submit.prevent="addCampaign">
-        <div class="medium-12 columns">
-          <label :class="{ error: $v.title.$error }">
-            {{ $t('CAMPAIGN.ADD.FORM.TITLE.LABEL') }}
-            <input
-              v-model.trim="title"
-              type="text"
-              :placeholder="$t('CAMPAIGN.ADD.FORM.TITLE.PLACEHOLDER')"
-              @input="$v.title.$touch"
-            />
-          </label>
-        </div>
+  <div class="column content-box">
+    <woot-modal-header
+      :header-title="$t('CAMPAIGN.ADD.TITLE')"
+      :header-content="$t('CAMPAIGN.ADD.DESC')"
+    />
+    <form class="row" @submit.prevent="addCampaign">
+      <div class="medium-12 columns">
```

Comment:
I think we can keep only one parent `medim-12.columns` and form inputs as children.